run_tests.sh

```diff
@@ -107,7 +107,8 @@ if [[ "$#" -eq 0 ]]; then
   # Note: Usually we want to use double-quotes on bash variables, however,
   # in this case we want a set of parameters instead of a single one.
   strip_path $files_list
-  run_tests
+  # Note: LANGUAGE environment variable must be set in order to avoid error messages in other languages.
+  LANGUAGE=en_US.UTF_8 run_tests
```

Comment:
You also need to do this in line 122 to avoid single test runs like

```bash
./run_tests.sh test kwlib_test
```

from failing.